src/components/core/DpDashboardTaskCard.vue

```diff
@@ -41,6 +41,21 @@ export default {
     procedureId: {
       type: String,
       required: true
+    },
+
+    resourceListRoute: {
+      type: String,
+      required: true
+    },
+
+    segmentListHashUpdateRoute: {
+      type: String,
+      required: true
+    },
+
+    segmentsListRoute: {
+      type: String,
+      required: true
```

Comment:
We should stay/go Topic agnostic

Reply:
changes of names done in https://github.com/demos-europe/demosplan-ui/pull/91/commits/ad1afe6b79581b0b0349773cfb2059ebc9083027

Reply:
@hwiem Done in https://github.com/demos-europe/demosplan-ui/pull/91/commits/d7d1be9918dbce8d6eb8dbf2a90682df845a9850

Reply:
Sorry for realizing only now, but do we even want to include that component in `demosplan-ui`? I'd say it's too specific.

It might be a good idea to finish the component cleanup in https://yaits.demos-deutschland.de/T30969 before fixing things that may later be removed from `demosplan-ui` entirely.

Reply:
Yes, we used this component only in one place `administration_dashboard.html.twig`!
What do I have to do now? Revert all only for this component?

Reply:
https://yaits.demos-deutschland.de/T30969#732881